test/server/graphql/v2/mutation/UpdateMutations.test.js

```diff
@@ -109,9 +110,59 @@ describe('server/graphql/v2/mutation/UpdateMutations', () => {
     });
   });
 
+  describe('create a changelog update', () => {
+    let user3, opencollective, changelogUpdate;
+    before(async () => {
+      sendEmailSpy.resetHistory();
+      user3 = await models.User.createUserWithCollective(utils.data('user3'));
+      opencollective = await models.Collective.create({ slug: 'opencollective' });
+      opencollective.addUserWithRole(user3, roles.ADMIN);
+      changelogUpdate = {
+        title: 'Monthly changelog update',
+        html: 'New feature added',
+        isChangelog: true,
+        account: { legacyId: opencollective.id },
+      };
+    });
+    const createUpdateMutation = gqlV2/* GraphQL */ `
+      mutation CreateUpdate($update: UpdateCreateInput!) {
+        createUpdate(update: $update) {
+          id
+          slug
+          publishedAt
```

Comment:
```suggestion
          publishedAt
          isChangelog
```

We should make sure the created update has `isChangelog` === true

Reply:
@Betree : Thanks for pointing it out. It's done. 😄